using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -735,7 +741,9 @@ ACPIdentity.getIdentifiers().then(identifiers => console.log("AdobeExperienceSDK
 
 This API retrieves the Experience Cloud ID that was generated when the app was initially launched and is stored in the Experience Cloud ID Service.
 
-This ID is preserved between app upgrades, is saved and restored during the standard application backup process, and is removed at uninstall. The values are returned via the [AdobeCallback](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/identity/identity-api-reference#adobecallback).
+This ID is preserved between app upgrades, is saved and restored during the standard application backup process, and is removed at uninstall. The values are returned via the `AdobeCallback`.
+
+When `AdobeCallbackWithError` is provided, a 500ms timeout is used when fetching the Experience Cloud ID from the Mobile SDK. If the operation times out or an unexpected error occurs, the `fail` method is called with the appropriate `AdobeError`.
```

Comment:
Please reword as follows:
"When `AdobeCallbackWithError` is provided, and you are fetching the ECID from the Mobile SDK, the timeout value is 500ms. If the operation times out or an unexpected error occurs, the `fail` method is called with the appropriate `AdobeError`."

(If this instance of "Experience Cloud ID" is 2nd instance or later on the page, we should change it to "ECID". Only the 1st instance should be spelled out as "Experience Cloud ID (ECID)".